### ISSUE TITLE
Action works only once

### ISSUE THREAD

**Issue by adamjcsmith** [segmentio/nightmare#814]
I implemented a custom action 'realClick' from [another issue](https://github.com/segmentio/nightmare/issues/639#issuecomment-219873334), and it works great the first time. Upon the second usage (using the same selectors, later in the test), nothing happens.

I've tried this out in a few different test suites, and just curious how this could be occurring.

``` javascript
  nightmare.action('realClick',
    function(name, options, parent, win, renderer, complete) {
      parent.respondTo('realClick', function(x, y, done) {
        win.webContents.sendInputEvent({
          type: 'mousedown',
          x,
          y,
          clickCount: 1,
        });
        win.webContents.sendInputEvent({
          type: 'mouseup',
          x,
          y,
        });
        setTimeout(done, 500);
      });
    complete();
  }, function(selector, done) {
    const child = this.child;
    this.evaluate_now(function(selector) {
      const bounds = document.querySelector(selector).getBoundingClientRect();
      return {
        left: bounds.left,
        top: bounds.top,
        width: bounds.width,
        height: bounds.height,
      };
    }, function(error, bounds) {
      if (error) return done(error);
      const x = bounds.left + bounds.width / 2;
      const y = bounds.top + bounds.height / 2;
      const cb = function() { done(); };
      child.call('realClick', x, y, cb);
    }, selector);
  });
```

**Comment by rosshinkley** [comment]
It looks like you're trying to call `.action()` on a Nightmare instance instead of the Nightmare prototype.  Is that the case?

Setting that aside, I suspect your custom action does not work the first time, either: the callback `cb` gets called with an error.  (It doesn't serialize nicely from the client code because `Error` isn't easily serializable, but that's left to  #646.)  Poking around in your client function, it turns out that the event being emitted is invalid - while the event name seems to be case-insensitive (documentation has them as `mouseDown` and `mouseUp`), the coordinates _must_ be integers.  If the height or width are even, you'll end up with a float, which Electron will reject.  Try using `Math.floor` to ensure your coordinates are round integers.
